test/unit/spec/v2/view-builder/views/SelectEnrollmentChannelOktaVerifyView_spec.js

```diff
@@ -75,4 +76,12 @@ describe('v2/view-builder/views/ov/SelectEnrollmentChannelOktaVerifyView', funct
     expect(testContext.view.model.get('authenticator.channel')).toEqual(QRCODE);
   });
 
+  it('Always have selected channel as email on mobile', function() {
```

Comment:
is the ios scenario already covered in unit?

Reply:
No.. i can add it.